NashvilleHistory/www/js/controllers/AllPlacesCtrl.js

```diff
@@ -18,14 +18,29 @@ app.controller('AllPlacesCtrl', function($scope, $state, $q, AllPlacesFact){
     )
     .then((data)=>{
       AllPlaces = data[0].concat(data[1]).concat(data[2]);
+      AllPlaces = AllPlaces.sort(sortAllPlaces);
       console.log("All places", AllPlaces);
       $scope.MarkerCards = AllPlaces;
-    })
+    });
+  }
+
+  function sortAllPlaces(x, y) {
+    // Determines if the place is a historical marker or artwork
+    // Returns the name of the place to be used when sorting
+    function getPlaceName(place) {
+      if (place.title) {
+        return place.title.toLowerCase();
+      } else if (place.artwork) {
+        return place.artwork.toLowerCase();
```

Comment:
I'll look look into it. I was trying to keep all of the `replace()` logic inside the factory, instead of having it in two places. I'll play around with it.